src/components/ToDoList/TodoWrapper.tsx

```diff
@@ -1,21 +1,29 @@
-import React from 'react';
-import { CustomTypeHooksType, TodoWrapperType } from './types';
+import React, { useEffect } from 'react';
+import { CustomTypeHooksType, TaskType } from './types';
 import TodoForm from './TodoForm';
 import Todo from './Todo';
 import useCustomTaskHooks from '../../hooks/CustomTaskHooks';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { getTodosAsync } from '../../features/todoList/TodosSlicer';
 
-function TodoWrapper({ todoTasks }: TodoWrapperType): JSX.Element {
-    const [tasks, addTask, deleteTodo, updateTask]: CustomTypeHooksType =
-        useCustomTaskHooks(todoTasks);
+function TodoWrapper(): JSX.Element {
+    const [addTask, deleteTask, updateTask]: CustomTypeHooksType =
+        useCustomTaskHooks();
+
+    const dispatch = useAppDispatch();
+    const { todos }: Array<TaskType> = useAppSelector((state) => state.todos);
```

Comment:
create selector in store, then useSelector(tasks)

Reply:
In [docs](https://redux-toolkit.js.org/tutorials/typescript) they recommend to create selector in a separate folder called hooks
Or I didn't get your instructions, what the actual problem?

Reply:
Okay, you can move them to the hooks fodder.